src/sap/handle_7.py

```diff
@@ -66,11 +67,22 @@ def extend_all_rykkerspaerrer_deadlines(session, row_indices):
 
 
 def get_fp_list(case_table):
-    """Get all unique fp-numbers from the table."""
+    """Get all unique fp-numbers from the table.
+    Exclude ones that has a value in the bilagsnummer column.
+    """
     fp_set = set()
     for row in range(case_table.RowCount):
         fp = case_table.getCellValue(row, 'ZZ_PARTNER')
         fp_set.add(fp)
+
+    # Remove any that has a value in the bilagsnummer column
+    for row in range(case_table.RowCount):
+        bilag = case_table.getCellValue(row, 'ZZOPBEL')
+        if bilag:
+            fp = case_table.getCellValue(row, 'ZZ_PARTNER')
+            if fp in fp_set:
+                fp_set.remove(fp)
```

Comment:
Would it not make sense to incorporate this in the previous for-loop? Something like

```
fp_set = set()
for row in range(case_table.RowCount):
    bilag = case_table.getCellValue(row, 'ZZOPBEL')
    if not bilag:
        fp = case_table.getCellValue(row, 'ZZ_PARTNER')
        fp_set.add(fp)
```